content/workload-reduction-toolkit/address-workload-issues/curriculum-planning-and-delivery/try-ideas-to-manage-planning.md

```diff
@@ -0,0 +1,87 @@
+        </li>
+        <li>
+        continuously reviewing and evaluating our systems in order to support all staff to achieve a healthy work life balance
+        </li>
+     </ul> 
+      </p>
+    </div>
+  </div>
+</div>
+
+## Background from Ruth Allen, Deputy Headteacher
+
+{inset-text}
+
+At King Charles I school we know that teaching is exhilarating and rewarding but recognise that it can also be exhausting. We understand that time is precious and that tired teachers do not make effective teachers. Our aim is for staff to not have to take work home. 
+{/inset-text}
+
+## 15 ways that we manage curriculum planning and delivery workload 
+
+Collaborative planning enables teachers to discuss and share best practice rather than everyone busily planning the same lessons in their individual classrooms. We accept that collaborative planning will lead to less autonomy in the classroom but the trade off is much less work to do at home. 
+
+Staff within departments, where possible, have preparation, planning and assessment time (PPA) together to allow for collaborative planning. 
+
+Departmental resources are shared centrally to stop individuals having to reinvent the wheel with new plans. 
+
+We provided better quality schemes of work that specify the sequence of key knowledge, deliberate practice to develop skills, a clear progression model, alongside homework tasks and resources to be used in lessons. This means there is no more endless searching for online resources for single activities, homework tasks and lesson ideas. 
+
+All teachers follow research driven teaching models that focus on activities that have a significant impact on progress. No more time spent creating time wasting teaching resources such as card sorts and slideshows.   
+
+We focus on the use of textbooks and ready-made quality assured resources to help reduce planning time and support better teaching. 
+
+No ‘all, most, some’ learning objectives or creation of different worksheets for different abilities. There is no need for any differentiation by task. 
+
+The PPA allocation for teachers is more than the requirements set out in the School Teachers’ Pay and Conditions Document and amounts to 12% of a teacher’s timetabled teaching hours. 
+
+There is no pressure to 'put on a show' in lessons: observations are not graded to help reinforce this. The number of lesson observations per year has been reduced.  
+
+There is no pointless requesting or completing of lesson plans. In fact, there is no need to write lesson plans of any kind.  
+
+There is no requirement to run lunch time or after school revision sessions. Instead, we focus on what is going on in lessons.  
+
+We creatively timetable with workload in mind. We aim for teachers to have not have classes from all year groups to reduce planning time and attendance at parents’ evenings.  
```

Comment:
Typos in this sentence - "We creatively timetable with workload in mind. Teachers generally don't have classes from all year groups, to reduce planning time and attendance at parents’ evenings."